src/components/main/AirQuality.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC, useState, useEffect, useContext } from 'react';
+
+import { getWeather } from 'services/getWeather';
+import { positionContext } from 'context/positionContext';
+
+import HourlySection from './HourlySection';
+
+import { AirQualityType } from 'types/airQualityType';
+
+const AirQuality: FC = () => {
+  const [airQuality, setAirQuality] = useState<AirQualityType[] | undefined>(undefined);
+  const {
+    state: { position }
+  } = useContext(positionContext);
+
+  useEffect(() => {
+    getWeather<{ forecast: AirQualityType[] }>(
+      '/air-quality/forecast/hourly/',
+      position.longitude,
+      position.latitude
```

Comment:
next time please refactor `getWeather` function to get `position` object instead of `position.longitude, position.latitude`:  1. it will be shorter; 2. longtitude and latitude are related to each other and not used separately